src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentalPrice = 40;
+  let total = 0;
+
+  if (days < 3) {
+    total = days * rentalPrice;
```

Comment:
Instead of doing calculation in multiple lines you can assign a variable to `days * rentalPrice`.